_posts/2020-09-11-liver-patient-classification/2020-09-11-liver-patient-classification.Rmd

```diff
@@ -0,0 +1,717 @@
+                                              "Total serum bilirubin (in mg/dL)",
+                                              "Direct bilirubin level (in mg/dL)",
+                                              "Serum alkaline phosphatase level (in U/L)",
+                                              "Serum alanine transaminase level (in U/L)",
+                                              "Serum aspartate transaminase level (in U/L)",
+                                              "Total serum protein (in g/dL)",
+                                              "Serum albumin level (in g/dL)",
+                                              "Albumin-to-globulin ratio",
+                                              "Target variable (1 = liver disease, 0 = no liver disease)"
+                                              )
+                              )
+
+kable(col_descr)
+```
+
+As one can see, besides age and gender, the dataset contains eight additional numerical features.
+While the names and corresponding measurements look rather cryptic to the uninformed eye, they are all part of standard blood tests conducted to gather information about the state of a patient's liver, so-called liver function tests.
+All of these measurements are frequently used markers for liver disease.
+For the first five, measuring the chemical compound bilirubin and the three enzymes alkaline phosphatase, alanine transaminase and aspartate transaminase, elevated levels indicate liver disease @Gowda2009 @Oh2011.
+For the remaining three, which concern protein levels, lower-than-normal values suggest a liver problem @Carvalho2018 @LabTests2019.
+Lastly, one should note that some of the measurements are part of more than one variable. For example, the total serum bilirubin is simply the sum of both the direct and indirect bilirubin levels and the amount of albumin is used to calculate the values of the total serum protein as well as the albumin-to-globulin ratio. So, one might already suspect that some of the features are highly correlated to one another, but more on that kind of analysis in the following section. 
+
+# Data exporation
+
+```{r overview, include=FALSE}
+## Data exploration
+str(liver_df)
+summary(liver_df)
+skim(liver_df)
+```
+
+## Missing values  
+
+After we got an overview of the task and dataset, we analyzed the variables from an exploratory perspective and first checked for missing values.
+
+```{r NAs, echo=FALSE}
+# Counting missing values by column
+countNA = sapply(liver, function(x) sum(is.na(x))) %>% data.frame()
+colnames(countNA) = "Number of NAs"
+countNA %>% kable()
+```
+```{r NAdetail, include=FALSE}
+liver_df[rowSums(is.na(liver_df)) > 0, ]
+```
+
+The only missing values, totalling four, are albumin-to-globulin ratios.
+As it turned out omitting or imputation techniques were not necessary, because we ended up not using the ratio due to high correlations with other features. 
```

Comment:
so this high correlation was observed above? Write a sentence in the section above then?